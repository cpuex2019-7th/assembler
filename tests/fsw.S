    li t1, 0x3f666666           ; 0.9
    fmvwx ft1, t1
    
    li sp, 0
    fsw ft1, sp, 0
    
    lw a0, sp, 0                ; a0 should be 0x3f666666
