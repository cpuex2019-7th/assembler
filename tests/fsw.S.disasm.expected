lui t1, 259686
addi t1, t1, 1638
fmvwx ft1, t1
addi sp, zero, 0
fsw ft1, sp, 0
lw a0, sp, 0
